lib/templ.ml

```diff
@@ -360,7 +377,6 @@ let apply_format conf nth s1 s2 =
                           match Util.check_format "%d%d" s1 with
                           | Some s3 -> Printf.sprintf s3 (int_of_string s21) (int_of_string s22)
                           | None -> "[" ^ s1 ^ "?]"
-  with _ -> s1
```

Comment:
I think we still need this try ... with
`Util.check_format` and `int_of_string` can fail

It would be nice to log this failure too